cradlepoint/pages/api/editResult.js

```diff
@@ -0,0 +1,33 @@
+const { ObjectId } = require('mongodb');
+import connectToDb from "../../util/mongodb";
+import {resultSchema} from "../../schemas/resultSchema";
+/*
+  Edits the requested result from the database
+*/
+
+export default async (req, res) => {
+  if (req.method !== 'PUT') {
+    res.status(405).send({ message: 'Only PUT requests allowed' })
+  }
+  try{
+    const data = req.body;
+    const valid = await resultSchema.isValid(data);
+    if (valid && ObjectId.isValid(data.testId)){
+      const result = resultSchema.cast(data);
+      // Set ID strings to Mongo ObjectId's
+      const newId = ObjectId(result._id);
+      const newTestId = ObjectId(result.testId);
```

Comment:
Similar to the other PR, can we name these just id and testId? I think the new is misleading